src/azul/service/manifest_service.py

```diff
@@ -613,9 +613,21 @@ def __init_subclass__(cls) -> None:
             assert format not in cls.cls_for_format
             cls.cls_for_format[format] = cls
 
+    @classmethod
+    def _cmd_exe_quote(cls, s: str) -> str:
+        """
+        Escape a string for insertion into a `cmd.exe` command line
+        """
+        assert '"' not in s, s
+        assert '\\' not in s, s
+        return f'"{s}"'
+
     @classmethod
     def command_lines(cls, url: str) -> Optional[JSON]:
-        return None
+        return {
+            'cmd.exe': f'curl.exe -L -O -J {cls._cmd_exe_quote(url)}',
+            'bash': f'curl -L -O -J {shlex.quote(url)}'
```

Comment:
```suggestion
            'bash': f'curl -L -OJ {shlex.quote(url)}'
```

Same for cmd.exe if that's possible. But test first.

I'm ok with using the short options but we need to add a comment explaining briefly (!) what each one does.